js/modules/k6/experimental/tracing/module.go

```diff
@@ -18,6 +18,9 @@ type (
 	// ModuleInstance represents an instance of the JS module.
 	ModuleInstance struct {
 		vu modules.VU
+
+		// Client holds the module's default tracing client.
+		*Client
```

Comment:
Is there a need for this field? It seems to me it is only used in one place so :shrug:

Reply:
Following your comment on throwing errors when `instrumentHTTP` is called twice, I now rely on this field to check if we have already initialized a module-wide client or not 👍🏻